Library/brew.sh

```diff
@@ -149,15 +150,19 @@ Otherwise, you should:
 EOS
   fi
 
-  XCRUN_OUTPUT="$(/usr/bin/xcrun clang 2>&1)"
-  XCRUN_STATUS="$?"
-
-  if [[ "$XCRUN_STATUS" -ne 0 && "$XCRUN_OUTPUT" = *license* ]]
+  # Don't check xcrun if the CLT isn't installed, as that causes a popup window
```

Comment:
I suppose that should have been something like “[…] if neither the CLT nor Xcode is installed […]”, right?